tests/conftest.py

```diff
@@ -75,6 +81,12 @@ def test_data() -> "dict[str, str]":
                 "functionType": "function/demo",
                 "configuration": {},
             },
+            # dataresource
+            "dataresource-910c9965-a318-4ac4-9123-9c55d5b86f2e": {
+                "downloadUrl": "https://filesamples.com/samples/code/json/sample.json",
```

Comment:
Are we mocking this URL response? Otherwise I would not use it as it has a tendency to return HTML as a first response - not JSON.